.github/workflows/rendersite.yml

```diff
@@ -4,14 +4,21 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v4
         with:
           java-version: '11'
           distribution: 'adopt'
-          cache: gradle
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          # Publish a Gradle build scan so we can see all the build logs,
+          # a complete task timeline, test outputs, and the resolved dependencies of the build.
+          build-scan-publish: true
+          build-scan-terms-of-use-url: "https://gradle.com/terms-of-service"
+          build-scan-terms-of-use-agree: "yes"
       - name: Render HTML for Site
```

Comment:
Add
```yaml
env:
GRADLE_ENTERPRISE_BUILD_CACHE_NODE_USER: ${{ secrets.GRADLE_ENTERPRISE_BUILD_CACHE_NODE_USER }}
GRADLE_ENTERPRISE_BUILD_CACHE_NODE_KEY: ${{ secrets.GRADLE_ENTERPRISE_BUILD_CACHE_NODE_KEY }}
```
for build cache authentication (used in settings.gradle)

Reply:
I'm going to assume mean I should add the `DEVELOCITY_BUILD_CACHE_NODE_USER` and `DEVELOCITY_BUILD_CACHE_NODE_KEY` environment variables just like in the `grails-data-mapping` repository workflows:

https://github.com/grails/grails-data-mapping/blob/f9e9f3e325f0dfccdbe464231af364449fc6c9ff/.github/workflows/groovy-joint-workflow.yml#L108-L110

Reply:
🤦 You were right. I just checked `settings.gradle` in this repository and now I see that it is indeed looking for the `GRADLE_ENTERPRISE_BUILD_CACHE_NODE_USER` and `GRADLE_ENTERPRISE_BUILD_CACHE_NODE_KEY` environment variables.